mlibrary/my_account
# From Other Institutions (Interlibrary Loan)

# Overview
This pull request adds ILLiad information to the [From Other Institutions (Interlibrary Loan)](http://localhost:4567/requests/interlibrary-loan) requests page.

## Empty State Container
For the case of no requests existing, an empty state container has been created to replace an empty table.

## Testing
* Run tests to make sure they pass (`docker-compose run web bundle exec rspec`). Break them to double-check.
* Build the CSS to make sure it works (`npm run build-css`).
* Check [From Other Institutions (Interlibrary Loan)](http://localhost:4567/requests/interlibrary-loan) requests page to see if the empty state appears.
* In `./models/interlibrary_loan_requests.rb`, replace `line 4` with:
  ```
  @requests = parsed_response.select { |request| request["ProcessType"] != "DocDel" && request["TransactionStatus"] == "Request Finished" }
  ```
  and then recheck the [From Other Institutions (Interlibrary Loan)](http://localhost:4567/requests/interlibrary-loan) requests page to see information appear (you may have to run `docker-compose restart`).
